package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "wp-scripts build --webpack-copy-php --webpack-src-dir=blocks",
     "create-block": "cd blocks && npx @wordpress/create-block --template ../bin/create-block --no-plugin",
-    "dev": "webpack --mode=development --watch",
+    "dev": "wp-scripts start --webpack-copy-php --webpack-src-dir=blocks",
```

Comment:
We should change this command to `start` to be the same as the `start` command from `@wordpress/scripts`.

Reply:
Since this script simply acts as a shorthand for running `wp-scripts start` we should name it as such.

Reply:
@stevenslack The `start` command is already available: https://github.com/alleyinteractive/create-wordpress-plugin/blob/fbd06fa948e863eebcb78b20672b655b42f66ed0/package.json#L31

The `dev` command is more of an alias.

Reply:
I just came here because I noticed the same thing!
If we want/need an alias that is fine, but we are repeating ourselves. Perhaps we could call the `start` script from the `dev` command `"dev": "npm run start",` or just remove the dev command.